src/engraving/libmscore/tie.cpp

```diff
@@ -839,10 +839,8 @@ void TieSegment::addLineAttachPoints()
     // Add tie attach point to start and end note
     Note* startNote = tie()->startNote();
     Note* endNote = tie()->endNote();
-    if (endNote && endNote->findMeasure() == startNote->findMeasure()) {
-        startNote->addLineAttachPoint(ups(Grip::START).pos(), tie());
-        endNote->addLineAttachPoint(ups(Grip::END).pos(), tie());
-    }
+    startNote->addLineAttachPoint(ups(Grip::START).pos(), tie());
+    endNote->addLineAttachPoint(ups(Grip::END).pos(), tie());
```

Comment:
Is the null-check no longer needed here?

Reply:
Good point. In principle, the null-check isn't strictly necessary, because startNote and endNote are never null. But I think it's safer if I null-check anyway, just in case we encounter a broken tie or something. I'll quickly fix this